server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -2126,6 +2126,11 @@ public boolean finalizeVirtualMachineProfile(final VirtualMachineProfile profile
             }
         }
 
+        String routerLogrotateFrequency = RouterLogrotateFrequency.valueIn(router.getDataCenterId());
+        s_logger.debug(String.format("The global setting [%s], with value [%s], will be used to configure the logrotate service frequency on the virtual router.",
```

Comment:
It's not from the global setting always, so better to log zone id as well.

```suggestion
        s_logger.debug(String.format("The setting [%s] with value [%s] for the zone [%s], will be used to configure the logrotate service frequency on the virtual router.",
```